listener_clients/m5stickc-listener/m5stickc-listener.ino

```diff
@@ -616,24 +616,23 @@ void evaluateMode() {
     }
     
     #if TALLY_EXTRA_OUTPUT
-    if (actualType == "preview") {
-      digitalWrite(led_program, HIGH);
-      digitalWrite (led_preview, LOW);
-      digitalWrite (led_aux, LOW);
-    } else if (actualType == "preview") {
-      digitalWrite(led_program, LOW);
+    if (actualType == "\"program\"") {
+      digitalWrite (led_program, LOW);
       digitalWrite (led_preview, HIGH);
-      digitalWrite (led_aux, LOW);
-    } else if (actualType == "aux") {
-      digitalWrite(led_program, LOW);
-      digitalWrite (led_preview, LOW);
       digitalWrite (led_aux, HIGH);
-    } else {
-      digitalWrite(led_program, LOW);
+    } else if (actualType == "\"preview\"") {
+      digitalWrite (led_program, HIGH);
       digitalWrite (led_preview, LOW);
+      digitalWrite (led_aux, HIGH);
+    } else if (actualType == "\"aux\"") {
+      digitalWrite (led_program, HIGH);
+      digitalWrite (led_preview, HIGH);
       digitalWrite (led_aux, LOW);
+    } else {
+      digitalWrite (led_program, HIGH);
+      digitalWrite (led_preview, HIGH);
+      digitalWrite (led_aux, HIGH);
     }
-    #endif
```

Comment:
This #endif should stay

Reply:
@mrsydster is correct. The #endif should stay.